docs/build/build-guides-template-basic.md

```diff
@@ -85,133 +92,215 @@ This tutorial won't go into the specifics of the template, nor will it go into t
+   "tokenSymbol": "UNIT"
+},
+"para_id": PARA_ID_HERE,
+"relay_chain": "rococo",
+```
+
+Once you finish modifying the file, it should look like this:
+
+```json
+{
+  "name": "Custom",
+  "id": "custom",
+  "chainType": "Live",
+  "bootNodes": [],
+  "telemetryEndpoints": null,
+  "protocolId": "my-live-protocol",
+  "properties": {
+    "ss58Format": 42,
+    "tokenDecimals": 12,
+    "tokenSymbol": "UNIT"
+  },
+  "para_id": YOUR_PARA_ID_HERE,
+  "relay_chain": "rococo",
+  "codeSubstitutes": {},
+  "genesis": { ... }
+}
+```
+
+Feel free to customize various aspects of your spec, such as the `UNIT` ticker, `name`, `id`, or
+other fields.
+
+Now you should open your `chain_spec.json`, and use this checklist to ensure all the necessary
+fields are in place:
+
+1. **Make** sure that `relay_chain` is set to the target relay chain (`rococo`, in our case)
+2. **Make** sure that `para_id` (right below `relay_chain`) is set to your reserved ParaId
+3. **Make** sure that our `chain_type` is set to `Live`
+4. **Optionally**, change the name, id, and token symbol of your chain.
+
+If you fail to do one of these, there is a large chance that your chain may fail to produce blocks.
+
+For more information on chain specifications,
+[check out the reference document from the Polkadot SDK.](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/chain_spec_genesis/index.html)
+
+### Generating the Runtime and Genesis
+
+With our chain specification successfully generated, we can move to generating the genesis state and
+runtime.
+
+Generate the genesis as follows:
+
 ```shell
-../../target/release/parachain-template-node export-genesis-state genesis
+polkadot-parachain export-genesis-head --chain chain_spec.json genesis
 ```
 
+Although you can use the WebAssembly within `wbuild`, for ease of access you can also regenerate
+your WebAssembly blob with the following:
+
 ```shell
-../../target/release/parachain-template-node export-genesis-wasm genesis-wasm
+polkadot-parachain export-genesis-wasm --chain chain_spec.json genesis-wasm
 ```
 
-Within `polkadot-sdk/templates/parachain`, you should now have two files:
+Within your project folder, you should now have two files:
 
 - **`genesis`** - the initial state of your parachain.
 - **`genesis-wasm`** - the initial runtime WebAssembly blob of your parachain.
 
 ## Running Your Collator
 
-It would help if you now started syncing your collator. Keep in mind that you will need to sync
-Rococo first - this could take some time (12 hours to a day - depending on your download speed), so
-best to get started ASAP. In order to avoid storing the full state of the relay chain, be sure to
-run with the appropriate pruning flags (`blocks-pruning` and `state-pruning`):
+> Make sure you have the
+> [`polkadot-parachain`](./build-guides-install-deps.md#installing-the-omninode) binary installed!
+
+You should start syncing your collator. Keep in mind that you will need to sync Rococo first - this
```

Comment:
This paragraph could improve with a bit of elaboration on the sync process. What do `blocks-pruning` and `state-pruning` do? even a small context on these flags will improve the reader's experience.

The tone is conversational and could be a bit more instructional. It's better to avoid terms like ASAP in technical documentation.

"could take some time depending on your download speed" -> "the sync process can benefit from higher network bandwidth as it is network resource intensive" or something along those lines